app/javascript/ui/casebooks/dragging.coffee

```diff
@@ -0,0 +1,97 @@
+ui = require 'ui/turbolinks'
```

Comment:
I would like to do ES6 --

Is babel necessary or can we just use sprockets-es6 for the ES6 syntax? 
webpacker is in rails 5 but that's probably more than we need?